pkg/tools/benchmark/operator/service/operator.go

```diff
@@ -248,22 +248,21 @@ func (o *operator) benchJobReconcile(ctx context.Context, benchJobList map[strin
 	// jobStatus is used for update benchmarkJob CR status if updating is needed.
 	jobStatus := make(map[string]v1.BenchmarkJobStatus)
 	for k := range benchJobList {
-		// update scenario status
 		job := benchJobList[k]
-		hasOwner := false
+		// update scenario status
 		if len(job.GetOwnerReferences()) > 0 {
-			hasOwner = true
-		}
-		if scenarios := o.getAtomicScenario(); scenarios != nil && hasOwner {
-			on := job.GetOwnerReferences()[0].Name
-			if _, ok := scenarios[on]; ok {
-				if scenarios[on].BenchJobStatus == nil {
-					scenarios[on].BenchJobStatus = map[string]v1.BenchmarkJobStatus{}
+			if scenarios := o.getAtomicScenario(); scenarios != nil {
+				on := job.GetOwnerReferences()[0].Name
+				if _, ok := scenarios[on]; ok {
+					if scenarios[on].BenchJobStatus == nil {
```

Comment:
Can we write this like below?
```go
if s, ok := scenarios[on];ok {
    s.BenchJobStatus =...
}
```

Reply:
@ykadowak 
I remember this being pointed out on Deepsource, but I'll try implementing it again.